FamilySearchIosSampleApp/LoginVC.swift

```diff
@@ -0,0 +1,244 @@
+//
+//  LoginVC.swift
+//  FamilySearchIosSampleApp
+//
+//  Created by Eduardo Flores on 6/3/16.
+//  Copyright © 2016 FamilySearch. All rights reserved.
+//
+
+import UIKit
+
+class LoginVC: UIViewController {
+    
+    @IBOutlet weak var usernameTextFiew: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var loginButtonOutlet: UIButton!
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    @IBOutlet weak var dataUsageLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        activityIndicator.stopAnimating()
+        
+        usernameTextFiew.placeholder = NSLocalizedString("usernamePlaceholderText", comment: "username, in email form")
+        passwordTextField.placeholder = NSLocalizedString("passwordPlaceholderText", comment: "password")
+        loginButtonOutlet.setTitle(NSLocalizedString("loginText", comment: "text for login button"), forState: UIControlState.Normal)
+        dataUsageLabel.text = NSLocalizedString("loginDataUsage", comment: "description of data usage")
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func loginAction(sender: AnyObject)
+    {
+        activityIndicator.startAnimating()
+        
+        let username = usernameTextFiew.text!
+        let password = passwordTextField.text!
+
+        if ( !(username.isEmpty) && !(password.isEmpty))
+        {
+            // get initial GET call to collections
+            Utilities.getUrlsFromCollections({ (collectionsResponse, error) -> Void in
+                if (error == nil)
```

Comment:
This is another great place where you could use a `guard` statement to keep the error check in the same place as the error handling:

```
guard error == nil else {
    print("Error getting collections data from server. Error = \(error?.description)")
    self.activityIndicator.stopAnimating()
    return
}